pkg/dao/repository.go

```diff
@@ -68,11 +71,17 @@ func (r repositoryDaoImpl) List(
 	}
 
 	if filterData.Arch != "" {
-		filteredDB = filteredDB.Where("arch = ?", filterData.Arch)
+		arches := strings.Split(filterData.Arch, ",")
+		filteredDB = filteredDB.Where("arch IN ?", arches)
 	}
 
 	if filterData.Version != "" {
-		filteredDB = filteredDB.Where("? = any (versions)", filterData.Version)
+		versions := strings.Split(filterData.Version, ",")
+		versionQuery := fmt.Sprintf("'%s' = any (versions)", versions[0])
+		for i := 1; i < len(versions); i++ {
+			versionQuery = versionQuery + fmt.Sprintf(" OR '%s' = any (versions)", versions[i])
+		}
+		filteredDB = filteredDB.Where(versionQuery)
```

Comment:
If you can think of a better way of comparing across two arrays, please let me know...

Reply:
This query works:

 select count(uuid) from repository_configurations where ARRAY['9', '8']::varchar[] @>  versions;

and might support something like:
```
filteredb.Where("ARRAY[?]::varchar[] @>  versions;", versions)
```
haven't tried i though :)